### PR TITLE
Add more codecs

### PR DESCRIPTION
**Pull request by kmilos**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -47,12 +47,18 @@ requirements:
     - jpeg
     - xz
     - zstd
+    - lerc
+    - libdeflate
+    - jbig
```
Comment (hmaarrfk) [inline recipe/meta.yaml:52]: Don't most of these have run exports?

Comment (kmilos) [reply recipe/meta.yaml:52]: Don't know, I just replicated what was done for other dependencies like `zlib`...

Comment (hmaarrfk) [reply recipe/meta.yaml:52]: Most of them have run exports. I cleaned things up. Thanks for adding this:

https://github.com/conda-forge/libwebp-base-feedstock/blob/master/recipe/meta.yaml#L14

**Comment by hmaarrfk** [comment]
Jbig needs to be migrated.

**Comment by hmaarrfk** [comment]
https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1511

**Comment by hmaarrfk** [comment]
Thanks for this.

**Pull request merged**